<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressDlg</class>
 <widget class="QDialog" name="ProgressDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Please Wait</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xiaoniao.qrc">
    <normaloff>:/xiaoniao/timgCADXKVUY.ico</normaloff>:/xiaoniao/timgCADXKVUY.ico</iconset>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>521</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Generating....</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="xiaoniao.qrc"/>
 </resources>
 <connections/>
</ui>
